src/interfaces/api/site.ts

```diff
@@ -107,6 +107,7 @@ export interface EditSite {
   application_question?: string;
   private_instance?: boolean;
   default_theme?: string;
+  legal_information?: string;
```

Comment:
What about createsite?

Reply:
I didnt add it there in the Lemmy PR, because it already has lots of fields and will probably not be used in lemmy-ui.